content/induction/blocks/cyf-folder/index.md

```diff
@@ -0,0 +1,31 @@
++++
+title = '🧰 Create a CYF folder'
```

Comment:
There's a video for this here https://www.youtube.com/watch?v=JFX8UfXWmok

You might want to pop the video in a tab?